### ISSUE TITLE
[Site Request] RokuHentai

### ISSUE THREAD

**Issue by howtopay** [manga-download/hakuneko#7127]
**Name of the website**
RokuHentai

**Website urls (examples below)**
- Site: https://rokuhentai.com/
- Manga List: https://rokuhentai.com/?q=
- Manga example: https://rokuhentai.com/f9ty6l/2
- Chapter online viewer example: https://rokuhentai.com/f9ty6l/2

**Languages**
The website in its self is english however multiple languages are available for each manga kinda like MangaDex, but where it is not sorted so there can be 4 manga's with the same title but different languages, whereas MangaDex makes them in one title but different languages in the chapters. 
I suggest making different connectors based on the language since it's easier to pick the right one for us users.

French
Russian
Spanish
Chinese
English
Japanese
Korean
Etc....

**Additional details**
When you click on a manga it doesn't have a description so directly to the chapter online viewer.

**Comment by MikeZeDev** [comment]
Too much pages : unknown number, may go for hours and hammering the server, even if they are JSON only.
TLDR : This will be clipboard only.

**Comment by MikeZeDev** [comment]
✅  Is added in [HaruNeko](https://docs.hakuneko.download/user-manual/)
🚫 Will not be added to HakuNeko

**Comment by jhonatanfish02** [comment]
Could you re-upload the manga by (Patreon) GabrAil?
